docs/components/connectors/out-of-the-box-connectors/microsoft-teams.md

```diff
@@ -95,17 +95,26 @@ For example, if you want to send a message in a Microsoft Teams channel, choose
+|     Members     |                                                                Create a new chat                                                                |   Yes    | FEEL expression |                                                               See [members property](#members-property) to learn more.                                                               |
+|       Top       |                                                              List messages in chat                                                              |    No    |     numbers     |                                                     Controls the number of items per response; maximum allowed top value is 50.                                                      |
+|    Order by     |                                                              List messages in chat                                                              |   Yes    |    dropdown     |                                                              Can order by 'lastModifiedDateTime' and 'createdDateTime'.                                                              |
+| Expand response |                                                                 Get chat by id                                                                  |   Yes    |    dropdown     |                                                                                        Choose                                                                                        |
+|     Filter      |                                                              List messages in chat                                                              |    No    |     string      | Sets the date range filter for the lastModifiedDateTime and createdDateTime properties. [Learn more about filtering](https://learn.microsoft.com/en-us/graph/filter-query-parameter) |
+|   Message ID    |                                                               Get message in chat                                                               |   Yes    |     string      |                                                                           Microsoft teams chat message id                                                                            |
+
+##### Expand response
+
+You can expand response for method **Get chat by ID**. For this in **Expand response** property you can :
+
+- choose **With chat members**, for getting information about members of chat;
+- choose **With last message preview**, for getting last message in chat. This function doesn't work for [client credentials type authentication](#client-credentials-type-authentication), make sure that you use another authentication type.
```

Comment:
- **With last message preview**, to get last message in chat. This function doesn't work with [client credentials type authentication](#client-credentials-type-authentication), make sure that you use another authentication type